pkg/reconciler/build/resources/pod_test.go

```diff
@@ -451,6 +463,23 @@ func TestMakePod(t *testing.T) {
 					},
 				},
 			)
+			c.want.InitContainers = append(c.want.InitContainers, entrypointContainer)
+			c.want.Volumes = append(c.want.Volumes, toolsVolume)
+			for i := range c.want.Containers {
+				c.want.Containers[i].Command = []string{"/tools/entrypoint"}
+				c.want.Containers[i].VolumeMounts = append(
+					c.want.Containers[i].VolumeMounts, toolsMount)
+				shouldWaitForPrevStep := false
+				if i > 0 {
+					shouldWaitForPrevStep = true
+				}
+				c.want.Containers[i].Env = append(
+					c.want.Containers[i].Env, corev1.EnvVar{Name: "ENTRYPOINT_OPTIONS",
+						Value: fmt.Sprintf(`{"args":null,"process_log":"/tools/process-log.txt","marker_file":"/tools/marker-file.txt","shouldWaitForPrevStep":%t,"preRunFile":"/tools/%d","shouldRunPostRun":true,"postRunFile":"/tools/%d"}`,
+							shouldWaitForPrevStep, i, i+1)},
+				)
+				c.want.Containers[i].Args = []string{}
+			}
```

Comment:
im guessing that writing this block of code made it easier to modify the expected test inputs and outputs without having to actually change the test cases - but i think for future readability and maintenance of these tests it'll be clearer to change the test cases themselves